javascript/react-js/react-introduction.md

```diff
@@ -6,14 +6,14 @@ This section aims to provide you with all you need to know to get started with R
 
 React is one of the most powerful, widely used frontend frameworks.
 
-The landscape for frontend frameworks has been changing a lot over the last few years, so it is understandable to be "scared" of going for the "wrong" one.
+The landscape for frontend frameworks has been changing a lot over the last few years, so it is understandable to be scared of going for the "wrong" one.
```

Comment:
```suggestion
The landscape for frontend frameworks has been changing a lot over the last few years, so it is understandable to be worried about choosing the "wrong" one.
```

This is just my opinion to avoid that term altogether and still convey the point.